Gemfile

```diff
@@ -17,7 +17,7 @@ group :test do
   gem 'rspec'
   gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.14.0')
   if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils', '~> 0.11'
+    gem 'sqa-utils', '0.12.1'
```

Comment:
@waynr, @nwolfe - This version of `sqa-utils` seem okay for testing from Puppet Server's stable branch - 2.x series / compatible with Puppet 4.2.Z series?  I'm totally ignorant about `sqa-utils`.